types/uid2/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {        
+        "dt-header": false
```

Comment:
Is there a reason not to enforce `dt-header`?

Reply:
This is similar to [another PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36211) I made for another npm module. 

For reference, here is the conversation from that other PR I made. This conversation goes over why I disabled this rule in this PR. 

![2019-07-18 at 9 32 AM](https://user-images.githubusercontent.com/2041082/61466204-1ad96780-a93f-11e9-90c2-af4ce72a8ff4.png)

I have made this same change in this PR now. I changed the version number for the uid2 package to 0.0 from 0.0.3 and then re-enabled the lint rule.